src/api/public/apidocs-new/paths/worker_cmd_checkconstraints.yaml

```diff
@@ -0,0 +1,114 @@
+post:
+  summary: Lists workers wich match a constraints filter.
+  description: |
+    Given a project, package, repository and architecture, list workers wich can build with that restrictions, and also match a constraints filter.
+
+    This endpoint doesn't create or modify any data.
+
+    More information about constraints can be found in the
+    [user guide](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.build_job_constraints.html).
+
+    This operation is the same as `POST /worker/command/run?cmd=checkconstraints`.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: project
+      schema:
+        type: string
+      required: true
+      example: home:user1
+      description: Project name.
```

Comment:
Once we merge #10972 you could use the extracted `project_name` parameters from [here](https://github.com/openSUSE/open-build-service/pull/10972/files#diff-8b3e66fe883e01e0b80fdef912a3ddff30318b8f92a6093de602a323e5768870R1)

Reply:
That doesn't fit in this case. This is a query parameter. If we change it from `project` to `project_name` it would not be recognized from the controller.

It happens the same with `repository_name` and `package_name`.

Reply:
Understood!